libs/MeshKernelApi/tests/src/ApiTest.cpp

```diff
@@ -3289,3 +3289,115 @@ TEST(Mesh2D, ConvertToCurvilinear_ShouldConvertMeshToCurvilinear)
     ASSERT_EQ(11, curvilinearOut.num_m);
     ASSERT_EQ(11, curvilinearOut.num_n);
 }
+
+TEST(Mesh2d, GetFacePolygons_OnAValidMesh_ShouldGetFacePolygons)
+{
+    int meshKernelId;
+    int isGeographic = 0;
+    meshkernelapi::mkernel_allocate_state(isGeographic, meshKernelId);
+
+    meshkernelapi::Mesh2D mesh2d;
+    std::vector<double> node_x{
+        0,
+        1,
+        2,
+        3,
+        0,
+        1,
+        2,
+        3,
+        0,
+        1,
+        3,
+        0,
+        1};
```

Comment:
I think if you add ; before }, you can move }; to a new line. Discovered that yesterday :)